src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -76,6 +76,8 @@ public static class EnvironmentSettingNames
         public const string FunctionsAlwaysReadyInstance = "FUNCTIONS_ALWAYSREADY_INSTANCE";
         public const string FunctionsTimeZone = "TZ";
         public const string FunctionsWebsiteTimeZone = "WEBSITE_TIME_ZONE";
+        public const string FunctionsTargetGroup = "FUNCTIONS_TARGET_GROUP";
+        public const string WebsiteArmResourceId = "WEBSITE_ARM_RESOURCE_ID";
```

Comment:
Please explain this new ARM ID thing - who is setting it on the app?

Reply:
Arjun is working on injecting this during specialization; this work is being tracked internally